sys-kernel/dracut/dracut-057-r3.ebuild

```diff
@@ -143,6 +143,15 @@ pkg_postinst() {
 		ewarn ""
 	fi
 
+	optfeature "gzip compression" app-arch/gzip
+	optfeature "bzip2 compression" app-arch/bzip2
+	optfeature "lzma compression" app-arch/lzma
+	optfeature "xz compession" app-arch/xz-utils
+	optfeature "lzo compression" app-arch/lzop
+	optfeature "lz4 compression" app-arch/lz4
+	optfeature "zstd compression" app-arch/zstd
+	optfeature "Parallel bzip2 compession" app-arch/lbzip2
+	optfeature "Parallel gzip compession" app-arch/pigz
```

Comment:
I would drop these two at least and replace `app-arch/gzip` → `app-alternatives/gzip` and `app-arch/bzip2` → `app-alternatives/bzip2`. Or maybe drop them all because it's not possible to have a system without these.